apps/docs/src/app/core/component-docs/tile/examples/kpi-tile-example.component.html

```diff
@@ -25,7 +25,7 @@ <h2 fd-tile-subtitle>
     </div>
     <div fd-tile-footer [twoColumn]="true">
         <div fd-tile-section>
-            <span fd-tile-refresh (click)="window.alert('Refresh clicked')" glyph="refresh"></span>
+            <i fd-tile-refresh (click)="window.alert('Refresh clicked')" glyph="refresh"></i>
```

Comment:
Looking ai this example I wonder how the user can click on this icon using only the keyboard.... I know it's not in the scope of this PR but we should def discuss it. Maybe also add aria-label for the screen readers